### DISCUSSION TITLE
NewsLetter and Engineering Blogs Feature

### DISCUSSION THREAD

**Discussion by aravindravva** [dailydotdev/daily#1022] [Feature Requests]
I had a similar idea of developing a website for developers focused on dev news, but your product is mature and already has 200,000+ users.

**Current Scenario** 
1. Several newsletters exist in the market for both development and personal growth. We could create a scraper to regularly extract data and post it on the site.
2.  problem with newsletters is that they are delivered to emails, and amidst other important content in emails, being busy with other tasks newsletters often go unnoticed.
3. Since there are numerous newsletters available, not everyone is aware of each one, leading to missed opportunities.
4. The news letters publishers spams the mails with the different promotion mails.

**proposed scenario**
1. Take the Link where the news letters archives are going to store.
2. scrape the website and store the title and summary.
3.  we can collect the different sources link from the users  and can categorize.  

I am open for contributions

**Comment by aravindravva** [comment]
https://github.com/dailydotdev/daily/issues/1021

**Comment by idoshamun** [comment]
Hi @aravindravva,

Thanks for the suggestion. We are already doing everything you mentioned.
Is there anything specifically that you miss?

**Reply by aravindravva** [reply]
it's on pipeline ? currently I can't see many popular  news letters

**Reply by aravindravva** [reply]
I want to work with you on this product :)

**Reply by idoshamun** [reply]
@aravindravva feel free to suggest newsletters that you find missing but we already have most of them
